rack-test.gemspec

```diff
@@ -1,14 +1,13 @@
 # -*- encoding: utf-8 -*-
 
-$LOAD_PATH.unshift File.expand_path('../lib', __FILE__)
-require 'rack/test/version'
+require_relative 'lib/rack/test/version'
 
 Gem::Specification.new do |s|
   s.name = 'rack-test'
   s.version = Rack::Test::VERSION
   s.platform = Gem::Platform::RUBY
-  s.author = 'Bryan Helmkamp'
-  s.email = 'bryan@brynary.com'
+  s.author = ['Jeremy Evans', 'Bryan Helmkamp']
+  s.email = ['code@jeremyevans.net', 'bryan@brynary.com']
```

Comment:
I personally advise we don't include email addresses in gemspecs. It can be another vector for security issues.

Reply:
I include my email address in all gems my maintain, and I'm basically the maintainer of rack-test at this point.  In my opinion, if your security depends on someone not being able to figure out your email address, that sounds like a problem to me.  Since it isn't your email address, do you still have a problem with it?